mappings/net/minecraft/client/network/ClientPlayerEntity.mapping

```diff
@@ -98,3 +98,4 @@ CLASS net/minecraft/class_746 net/minecraft/client/network/ClientPlayerEntity
 		ARG 2 dz
 	METHOD method_3149 isAutoJumpEnabled ()Z
 	METHOD method_3151 getMountJumpStrength ()F
+	METHOD method_33689 onRespawnOrDimensionChange ()V
```

Comment:
This is more like `init` or `setup` as it sets up the position, velcity, and pose of the player. Also it initializes health and sets deathTime to 0 which I think prevents the player from rendering as "dying".
```suggestion
	METHOD method_33689 init ()V
```